src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/EditApprenticeshipResponse.cs

```diff
@@ -0,0 +1,10 @@
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Api.Types.Responses
+{
+    public class EditApprenticeshipResponse
+    {
+        public long ApprenticeshipId { get; set; }
+        public bool NeedReapproval { get; set; }
```

Comment:
Would `RequireApproval` be a better name for this property?

Reply:
Yes it does, I will make this change on CON-3301, will save me from creating two packages.